### DISCUSSION TITLE
Make the repo for a package more prominent

### DISCUSSION THREAD

**Discussion by cassidoo** [npm/feedback#50] [Website] [answered]
**Problem Statement**
A grand majority of the time I open a package on npm, I look at the repo before installing anything. The graphs are nice and the README is nice (very excited about #4 btw), but I want to see how the code is structured and examples of it being used. Right now the repo and homepage link are fine, but the fact that the downloads graph, size, license, and other stats are above them doesn't really help me at all.

**Potential solution**
Either putting those links above the stats, or even just moving them to an area on the top (like in a tab or underneath the package name) would be really nice. It's minor and would only save a second or two of scrolling, but it really is something that I check for nearly every package.

**Comment by MylesBorins** [comment]
Hey @cassidoo thanks so much for the suggestion. I did a quick mock up, lmk if this is what you are thinking about (obviously needs more thought re: design + mobile layout).

While this is unrelated to your ask you can do `npm repo ${packageName}` to open the repo in your browser.

<img width="1766" alt="Screen Shot 2020-10-01 at 4 41 47 PM" src="https://user-images.githubusercontent.com/498775/94861385-71787780-0405-11eb-9af3-e2b20555c87e.png">

**Reply by ljharb** [reply]
this is great! it might look nice as well if the two links were floated right?

**Reply by Ethan-Arrowood** [reply]
Thats a nifty cli command! I gotta try using that in the future

**Comment by cassidoo** [comment]
This design is exactly what I had in mind!! Nothing too fancy, just easy-to-see links. I love it. 😊

**Reply by MylesBorins** [reply]
hey @cassidoo it took a hot minute but we shipped an improvement here. Links are moved up higher in the sidebar

**Comment by MylesBorins** [comment, marked as answer]
We have a tracking issue to explore doing this work

https://github.com/npm/roadmap/issues/8
